docs/sources/tempo/setup/deployment.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Plan your Tempo deployment
-menuTitle: Plan your deployment
+menuTitle: Plan your Tempo deployment
```

Comment:
In this case, we're not using Tempo in the menu title since we're in Tempo documentation. It should be obvious to the user that this is for Tempo deployment. The title: element should actually be updated to remove Tempo, as well as the page title.  
```suggestion
menuTitle: Plan your deployment
```